lib/kdump_utils.pm

```diff
@@ -169,7 +169,7 @@ sub activate_kdump {
     if (is_ppc64le || is_aarch64) {
         send_key('alt-y');
         type_string $memory_kdump;
-        send_key('ret');
+        send_key_until_needlematch('yast2-kdump-restart-info', 'ret');
```

Comment:
I would suggest to avoid having the possibility to trigger two `returns` in case the needle doesn't match for some reason...it might press the next OK accidentally, deleting the field before typing on it might make the trick, for example.

Reply:
It seems like there was some UI change that makes it necessary to hit enter twice: https://openqa.suse.de/tests/8257437#step/yast2_kdump/19 we enter 320, which is more than the maximum allowed value. So Yast changes it to the max allowed value, 287, after we type enter, but then we need to type enter again. Another option would be to set $memory_kdump to 287 only for aarch64, but that might cause problems again if the value changes for some reason. Typing enter twice does not hurt in this case.